wiki/Help_centre/Tournament_bans/en.md

```diff
@@ -0,0 +1,57 @@
+# Tournament bans
+
+*Main page: [Help centre](/wiki/Help_centre)*
+
+A tournament ban prevents a player from being involved in officially supported tournaments in any capacity. This includes playing and staffing or any other kind of assistance.
+
+Tournament bans are very rarely handed out their own, but generally speaking almost all players who return under a standard appeal for an [account restriction](/wiki/Help_centre/Account_restrictions) are subject to one of at least a year in length.
+
+Due to the high-stakes nature of tournament play, tournament bans are treated very seriously and are not subject to the kind of leniency an account restriction might ordinarily receive.
+
+## What causes someone to receive a temporary tournament ban?
+
+Any severe violation of the [community rules](/wiki/Rules) that takes place during an [officially supported tournament](/wiki/Tournaments/Official_support) is likely to cause a tournament ban, even if it would have typically only resulted in an infringement outside of the competitive play setting.
+
+Some real-world examples of such behavior is as follows:
+
+- Drawing hate symbols (swastika, etc.) using cursor smoke in streamed matches
+- Writing slurs or parts of slurs in cursor smoke in streamed matches
```

Comment:
Remove the first of these as it's not adding much and basically the same as the second?